storage/scylladb_Permanode.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef STORAGE_SCYLLADB_PERMANODE_H_
+#define STORAGE_SCYLLADB_PERMANODE_H_
```

Comment:
It is weird to name `scylladb_Permanode.h`. Can you revise?

Reply:
There's already a file named `scylladb_permanode.h` before and I delete it in this PR.
I would like to name `scylladb_permanode.h` too. But doing this will make this PR much harder to be reviewed since this file will be treated as modifying from the deleting `scylladb_permanode.h`

I think I could rename it to `scylladb_permanode.h` when we are going to merge this PR.